code/modules/mob/emote.dm

```diff
@@ -46,3 +46,28 @@
 					riding_datum.force_dismount(M)
 			else
 				R.unbuckle_all_mobs()
+
+/datum/emote/clap
+	key = "clap"
+	key_third_person = "claps"
+	message = "claps."
+	muzzle_ignore = TRUE
+	restraint_check = TRUE
+	mob_type_allowed_typecache = list(/mob/living/carbon/human, /mob/dead/observer)
+	mob_type_ignore_stat_typecache = list(/mob/dead/observer)
+
+/datum/emote/clap/run_emote(mob/user, params)
+	. = ..()
+	if (.)
+		var/clap = pick('sound/misc/clap1.ogg',
+				        'sound/misc/clap2.ogg',
+				        'sound/misc/clap3.ogg',
+				        'sound/misc/clap4.ogg')
+		if (ishuman(user))
```

Comment:
iscarbon?

Reply:
No, simple mobs don't clap.

Reply:
I'm envisioning a goliath flailing its tentacles in the air and slapping them together.

Reply:
@MoreRobustThanYou not sure if that's a good idea

Reply:
If that isn't then ghosts with literally nothing but emote memeing don't need this either?

Reply:
you guys still let this sneaky autistic shit code here? he messed up hippie gangs covertly and did horribly convoluted code

Reply:
he's actually an admin here.

Reply:
@zamolxius who are you talking about?
MRTY?

Reply:
@zamolxius gangs was shit